bokeh/charts/_charts.py

```diff
@@ -144,17 +147,28 @@ def start_plot(self, xgrid, ygrid):
         if ygrid:
             self.make_grid(1, yaxis.ticker)
 
-        # Add tools
+        # Add tools if supposed to
         if self.tools:
+            # only create pan and zoom tools if chart is not categorical...
+            if self.categorical:
+                tools_to_remove = ['pan', 'zoom']
+            else:
+                tools_to_remove = []
```

Comment:
> @damianavila That is a different case.. I check None just to set empty list as a default value to make https://github.com/bokeh/bokeh/pull/1624/files#diff-3e6325a902133d8fd2e886fe13214143R362 work without other checks

ok... but I am not saying it is the same case, just saying that you can put `None` here and the result will be the same (or I am missing something?)... but if you want to leave the empty list, it is ok too... np.